spec/rspec/core/formatters/html_snippet_extractor_spec.rb

```diff
@@ -15,7 +15,7 @@ module Formatters
         it "falls back on a default message when it gets a security error" do
           message = nil
           with_safe_set_to_level_that_triggers_security_errors do
-            message = RSpec::Core::Formatters::HtmlSnippetExtractor.new.lines_around("blech", 8)
+            message = RSpec::Core::Formatters::HtmlSnippetExtractor.new.lines_around("blech".taint, 8)
           end
```

Comment:
Suggestion: instead of assigning `message` to `nil` and then mutating it in the block...why not ensure `with_safe_set_to_level_that_triggers_security_errors` returns the value the block itself returns, and then just do:

``` ruby
message = with_safe_set_to_level_that_triggers_security_errors do
  RSpec::Core::Formatters::HtmlSnippetExtractor.new.lines_around("blech".taint, 8)
end
```

Seems simpler, right?

(Not a merge blocker, obviously).